client/src/components/Bills/BillList.js

```diff
@@ -0,0 +1,144 @@
+import React from "react";
+import { useState, useEffect } from "react";
+import BillForm from "./BillForm";
+
+const emptyBill = {
+  transaction_date: "",
+  subtotal: null,
+  tax_rate: null,
+  tax_amount: null,
+  tip_rate: null,
+  tip_total: null,
+  who_paid: null,
+  paid_up: null,
+  bill_notes: "",
+  full_total: null,
+};
+
+function BillList() {
+  //This needs an empty array, or the whole thing breaks
+  const [bills, setBills] = useState([]);
+  const [editBillById, setEditBillById] = useState(null);
+
+  useEffect(() => {
+    fetch("http://localhost:5005/db/bills")
+      .then((response) => response.json())
+      .then((billData) => {
+        setBills(billData);
+      });
+  }, []);
+
+  //  const deleteP Contacts
+  const deleteBill = async (bill_id) => {
+    try {
+      const deleteResponse = await fetch(
+        `http://localhost:5005/db/bills/${bill_id}`,
+        {
+          method: "DELETE",
+        }
+      );
+      if (deleteResponse.status === 200) {
+        setBills(bills.filter((bill) => bill.bill_id !== bill_id));
+      }
+    } catch (error) {
+      console.error(error);
+    }
+  };
+
+  //PUT stuff
+  // grabs the id of the post to be editted
+  const editBill = (bill) => {
+    const editId = bill.bill_id;
```

Comment:
Below you are calling:
```
onClick={() => editBill(bill.bill_id)}
```

So I think what's getting put in here is the `bill_id` and not the `bill` so l think `const editId = bill.bill_id` is unnecessary?

Reply:
Changed it so it is now: 
const editBill = (billId) => {
    console.log(billId);
    setEditBillById(billId);
  };

the thing being passed in is one of those instances, where it doesn't matter what it's called, it's just the value being passed in with the button.